app/views/koi/nav_items/_nav_item.html.erb

```diff
@@ -58,6 +58,12 @@
               </div>
             <%- end -%>
 
+            <%- if nav_item.read_attribute(:link_to_first_child) -%>
```

Comment:
Any reason this isn't `nav_item.link_to_first_child?`? 

There are few valid use cases for `read_attribute`.  It's useful when you've overridden a field with a method and you need to get the original value. I can't see a reason to use it here—correct me if I'm missing something!

Reply:
(I've tested it with `nav_item.link_to_first_child?` and it seems to work fine.)

Reply:
@Obversity I probably copied and pasted it from somewhere else in the project. 
Pretty standard when the front-end dev has to do back-end work :P 
I'm happy for it to be changed to whatever is more efficient, best-practice or future-proof. 
That's what code reviews are good for.

Reply:
@Obversity The same thing is here:
https://github.com/katalyst/koi/blob/ebe77fded78befb5e25058829827d9a24973582e/app/views/koi/nav_items/_nav_item.html.erb#L6

Want me to change it there too? 
I'm not going to pretend I know the difference between the two methods or why one be used over the other. 
Typically I assume this would be written as `nav_item.is_hidden.eql?(true)` or even `nav_item.is_hidden?` which seems to be a valid NavItem method. 

Let me know which way you want to go with this one.

Reply:
>  I probably copied and pasted it from somewhere else in the project. Pretty standard when the front-end dev has to do back-end work.

I figured that'd be the case!

Use `nav_item.link_to_first_child?` and `nav_item.is_hidden?` in those cases. (Let me know if you'd rather I did the commit(s), for accountability's sake.)